frontend/src/metabase/static-viz/containers/IsomorphicStaticChart/types.ts

```diff
@@ -7,4 +7,5 @@ export interface IsomorphicStaticChartProps {
   renderingContext: RenderingContext;
   width?: number;
   height?: number;
+  isStorybook?: boolean;
```

Comment:
it is a nitpick but ideally, production code should not be aware of tools we use for development, so maybe a better name would be `isGraalVM`

Reply:
I think it's okay, we use `isCypressActive` in other parts of prod code. Plus an optional `true` boolean is more elegeant since we can just add the prop name to the component, rather than `isGraalVM={false}`